pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[tool.poetry]
+name = "ngboost"
+version = "0.3.8dev"
+description = "Library for probabilistic predictions via gradient boosting."
+authors = ["Stanford ML Group <avati@cs.stanford.edu>"]
+readme = "README.md"
+homepage = "https://github.com/stanfordmlgroup/ngboost"
+repository = "https://github.com/stanfordmlgroup/ngboost"
+documentation = "https://stanfordmlgroup.github.io/ngboost/intro"
+classifiers = [
+    "Operating System :: OS Independent"
+]
+license = "Apache License 2.0"
```

Comment:
feel free to verify all the information here is correct, I ported things over from `setup.py` :)

Reply:
By removing ngboost/version.py will make package and make publish aquire the version number form here? For example after merging I would need to bump the version down here to 0.3.7 before running make pacakage and make publish.

Reply:
yep, correct - hopefully we can move to a process in the future were releasing a package version is done by:

* Creating a PR that merges to master with a bump in the version
* On master, GH action runs that sees a changed version and releases to pypi